File: python/prism_sdk/throughput_projection_integrity_workflow_fabric.py
"""Graph P32 throughput workflow_fabric projection-integrity feature F15."""
from .projection_integrity_support import ProjectionRequest4,ProjectionCard7,ProjectionIntegrityError,manifest,qualify
FEATURE_ID="AFA-graph-P32-F15";CONTRACT_VERSION="graph-throughput_projection_integrity_workflow_fabric/1.0"
def throughput_projection_integrity_workflow_fabric_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="workflow_fabric")
def qualify_throughput_projection_integrity_workflow_fabric(request:ProjectionRequest4)->ProjectionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="workflow_fabric")

File: crates/graph/src/throughput_projection_integrity_workflow_fabric.rs
//! Graph P32 throughput workflow_fabric projection-integrity feature.
use super::projection_integrity_support::{manifest,qualify,ProjectionCard7,ProjectionIntegrityError,ProjectionRequest4};
pub const FEATURE_ID:&str="AFA-graph-P32-F15";pub const CONTRACT_VERSION:&str="graph-throughput_projection_integrity_workflow_fabric/1.0";
pub fn throughput_projection_integrity_workflow_fabric_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","workflow_fabric")}
pub fn qualify_throughput_projection_integrity_workflow_fabric(request:&ProjectionRequest4)->Result<ProjectionCard7,ProjectionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"throughput","workflow_fabric")}
